github/lib/CreateIssueStatsBar.ts

```diff
@@ -0,0 +1,44 @@
+import { BlockBuilder } from "@rocket.chat/apps-engine/definition/uikit";
+import { IGitHubIssue } from "../definitions/githubIssue";
+import { OcticonIcons } from "../enum/OcticonIcons";
+
+export async function CreateIssueStatsBar(
+    issueInfo : IGitHubIssue,
+    block : BlockBuilder
+){
+    block.addContextBlock({
+        elements: [
+            block.newImageElement({
+                imageUrl: OcticonIcons.COMMENTS,
+                altText: "Comments",
+            }),
+            block.newPlainTextObject(
+                `${issueInfo.comments}`,
+                false
+            ),
+            block.newImageElement({
+                imageUrl: OcticonIcons.ISSUE_OPEN,
+                altText: "Assignees Icon",
+            }),
+            block.newPlainTextObject(
+                issueInfo.assignees ? (issueInfo.assignees.length == 0
+                    ? "No Assignees"
+                    : `${issueInfo.assignees.length} Assignees`) : ""
+            ),
+            block.newImageElement({
+                imageUrl:
+                    issueInfo.state == "open"
+                        ? OcticonIcons.ISSUE_OPEN
+                        : OcticonIcons.ISSUE_CLOSED,
+                altText: "State",
+            }),
+            block.newPlainTextObject(`${issueInfo.state}`),
+            block.newImageElement({
+                imageUrl: issueInfo.user_avatar ?? "",
+                altText: "User Image",
+            }),
+            block.newPlainTextObject(`Created by ${issueInfo.user_login}` ?? ""),
```

Comment:
@henit-chobisa  `??` operator is new for me, thanks  :) Are there any advantages to using this over  using simple `||` ? If there is no specific case where this helps, I think we can change it to `||` . Not a suggested change. Just curious

Reply:
Ohh I see, I am a bit habitual to this one actually replacing null values using the succeeding string, 
Actually both the operators kinda work the same way, logically opposite of each other 🐶